build.gradle

```diff
@@ -431,7 +431,7 @@ project(':iceberg-spark-runtime') {
     relocate 'org.apache.orc', 'org.apache.iceberg.shaded.org.apache.orc'
     relocate 'io.airlift', 'org.apache.iceberg.shaded.io.airlift'
 
-    archiveName = "iceberg-spark-runtime-${version}.${extension}"
+    archiveBaseName = "iceberg-spark-runtime"
```

Comment:
Is this needed? It seems like these are two ways to do the same thing?

Reply:
This maintains compatibility for future gradle upgrades because archiveName is deprecated. see: https://docs.gradle.org/current/dsl/org.gradle.api.tasks.bundling.Jar.html#org.gradle.api.tasks.bundling.Jar:archiveName

Reply:
Looks like this didn't work. The shadow Jar is now getting created with a classifier, `-all`. This is supposed to replace the original Jar so that the only Jar produced is the shaded Jar since there are no classes in this module.

I'm going to roll this back as part of the licensing work, but feel free to submit a follow-up PR that fixes it.